### PR TITLE
Task video hsl

### PR DESCRIPTION
**Pull request by rishi9684**
#video player added
#poster image added

### CODE REVIEW

File: pages/apps/_app/virtual/_id/index.vue [hunk 1]
```diff
@@ -14,7 +14,10 @@
             :key="item.id"
             class="xs12 sm4 md4 lg4 boxviewsmall pa-3 my-1 mx-0 py-2 session-view-in"
             :class="{
-              selected: item.WebinarLink + '?autoplay=1' === videoSrc,
+              selected:
+                `https://live.bitpod.io/hls/${
```
Comment (SukantGujar) [inline pages/apps/_app/virtual/_id/index.vue:18]: Should not be hardcoded.

File: pages/apps/_app/virtual/_id/index.vue [hunk 2]
```diff
@@ -252,7 +265,9 @@
                           class="xs12 sm4 md4 lg4 grey lighten-2 boxviewsmall pa-3 mb-4 mx-0 py-2 session-view-in"
                           :class="{
                             selected:
-                              item.WebinarLink + '?autoplay=1' === videoSrc,
+                              `https://live.bitpod.io/hls/${
```
Comment (SukantGujar) [inline pages/apps/_app/virtual/_id/index.vue:268]: Should not be hardcoded.

File: pages/apps/_app/virtual/_id/index.vue [hunk 3]
```diff
@@ -757,10 +775,21 @@ export default {
       }, 2000)
     },
     videoTileClick(item) {
-      this.videoSrc = item.WebinarLink + '?autoplay=1' || ''
+      this.videoSrc =
+        `https://live.bitpod.io/hls/${
```
Comment (SukantGujar) [inline pages/apps/_app/virtual/_id/index.vue:779]: Should not be hardcoded

**Review by SukantGujar — CHANGES_REQUESTED** [review]

File: pages/apps/_app/virtual/_id/index.vue (hunk 3 above)
Comment (rishi9684) [reply pages/apps/_app/virtual/_id/index.vue]: added link in public config

File: pages/apps/_app/virtual/_id/index.vue (hunk 2 above)
Comment (rishi9684) [reply pages/apps/_app/virtual/_id/index.vue]: added

File: pages/apps/_app/virtual/_id/index.vue (hunk 1 above)
Comment (rishi9684) [reply pages/apps/_app/virtual/_id/index.vue]: added link in public config

File: nuxt.config.js [hunk 4]
```diff
@@ -252,6 +263,7 @@ export default {
     cdnUri:
       'https://res.cloudinary.com/mytestlogo/image/upload/bitpodjs/images/',
     cdnCsvUri: 'https://res.cloudinary.com/mytestlogo/raw/upload/',
+    rtmpLink: 'https://live.bitpod.io/hls/',
```
Comment (SukantGujar) [inline nuxt.config.js:266]: Check with Yash on where to place these keys so they are picked up from env correctly.

**Review by SukantGujar — CHANGES_REQUESTED** [review]

File: nuxt.config.js (hunk 4 above)
Comment (rishi9684) [reply nuxt.config.js]: added process env

**Review by SukantGujar — APPROVED** [review]

**Pull request merged**